types/hapi/catbox/index.d.ts

```diff
@@ -0,0 +1,249 @@
+// TODO needs tests
+// TODO move to own definition
+// Type definitions for catbox 7.1
+// Project: https://github.com/hapijs/catbox
```

Comment:
these all seem to be valid node modules. they should live in top-level folders and not under `hapi`